sencha-workspace/SlateAdmin/app/controller/People.js

```diff
@@ -253,7 +255,7 @@ Ext.define('SlateAdmin.controller.People', {
                 }
 
                 me.selectPerson(person, function() {
-                    ExtHistory.resumeState();
+                    ExtHistory.resumeState(false);
```

Comment:
@BillClinton why add the false here? What wasn't working? I'm concerned there was probably a specific reason it wasn't there before